code/parse/sexp.cpp

```diff
@@ -12403,6 +12410,50 @@ void sexp_set_player_orders(int n)
 	}
 }
 
+void sexp_set_order_allowed_target(int n)
```

Comment:
I think you should add the for here.  The meaning is a little too ambiguous here without, imho.

Reply:
I'd bet money that I thought about typing the "for" and it just never made it to the keyboard... Fixed.